src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -71,6 +71,7 @@ public class RoatpApplicationPreambleController : RoatpApplyControllerBase
         }
 
         [Route("conditions-of-acceptance")]
+        [HttpGet]
         public async Task<IActionResult> ConditionsOfAcceptance(SelectApplicationRouteViewModel routeViewModel)
```

Comment:
Remove async Task from the signature as there are no awaited task in the method.

Reply:
Done